config/projects/angrychef.rb

```diff
@@ -47,6 +47,17 @@
 compress :dmg
 
 package :msi do
+  install_path = Pathname.new(install_dir)
+
+  # Find path in which the Chef gem is installed
+  search_pattern = "#{install_path}/**/gems/chef-[0-9]*-mingw32/ext/win32-eventlog/chef-log.dll"
+  dll_path  = Pathname.glob(search_pattern).find { |path| path.file? }
```

Comment:
And on second thoughts - schisamo did point out that we already have some fairly specific helper methods in the MSI packager.  I'll refactor this (and push-jobs).